algorithms/CPlusPlus/Trees/print-all-nodes-at-a-level.cpp

```diff
@@ -0,0 +1,61 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+typedef struct Node
```

Comment:
please provide a description of the program

Reply:
I have added a description.